### PR TITLE
Pioneer 18 deploy csv updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
P18 Leg 2 updates.

- Use attached mooring component depths Ss to check PM sensor depths.

- Use tables provided by Andrew in data slack channel to check deploy/recovery dates/times, lat/lon, depths.  

- All PMs except PMUI only need deploy/recovery info checked (rest of info already checked in pre-cruise PR).  PMUI-17 deploy needs everything checked. Per Al, WFP 13103-03 was recovered from OSPM-15 and hot swapped for PMUI-17 deployment (check via config; WFP sensor SNs can be confirmed via OSPM-15 deploy csv)

- Also includes 2 updates to SM recoveries done on Leg 1 - CNSM & ISSM - corrected to MFN release times.  Leg 1 QLR currently being reviewed, but can use that to check updated recovery table. (in Coastal Pioneer Cruises/_Pioneer Cruise Docs/3204-01802).

[CGSN_Mooring_Component_Depths.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/8566157/CGSN_Mooring_Component_Depths.xlsx)

### CODE REVIEW

**Review by snwhite — APPROVED** [review]

File: deployment/CP01CNSM_Deploy.csv [hunk 1]
```diff
@@ -475,40 +475,40 @@ AR52,,AR61,,CP01CNSM-SBD12-04-PCO2AA000,14,1,2021-03-31T15:36:00,2021-11-03T17:0
...
+AR61,,AR66,,CP01CNSM-SBD11-06-METBKA000,15,1,2021-11-03T14:45:00,2022-04-13T12:05:00,CGMCP-01CNSM-00015,,CGINS-METLGR-00007,40.139823,-70.771173,,0,132,,,
+AR61,,AR66,,CP01CNSM-SBD12-00-DCLENG000,15,1,2021-11-03T14:45:00,2022-04-13T12:05:00,CGMCP-01CNSM-00015,,CGCON-BDCL12-00009,40.139823,-70.771173,,0,132,,,
+AR61,,AR66,,CP01CNSM-SBD12-03-HYDGN0000,15,1,2021-11-03T14:45:00,2022-04-13T12:05:00,CGMCP-01CNSM-00015,,CGINS-HYDGN0-49860,40.139823,-70.771173,,0,132,,,
+AR61,,AR66,,CP01CNSM-SBD12-04-PCO2AA000,15,1,2021-11-03T14:45:00,2022-04-13T12:05:00,CGMCP-01CNSM-00015,,CGINS-PCO2AA-33177,40.139823,-70.771173,,0,132,,,
+AR61,,AR66,,CP01CNSM-SBD12-05-WAVSSA000,15,1,2021-11-03T14:45:00,2022-04-13T12:05:00,CGMCP-01CNSM-00015,,CGINS-WAVSSA-04571,40.139823,-70.771173,,0,132,,,
+AR61,,AR66,,CP01CNSM-SBD12-06-METBKA000,15,1,2021-11-03T14:45:00,2022-04-13T12:05:00,CGMCP-01CNSM-00015,,CGINS-METLGR-00003,40.139823,-70.771173,,0,132,,,
+AR61,,AR66,,CP01CNSM-SBD12-08-FDCHPA000,15,1,2021-11-03T14:45:00,2022-04-13T12:05:00,CGMCP-01CNSM-00015,,CGINS-FDCHPB-03904,40.139823,-70.771173,,-5,132,,,
```
Comment (cdobs) [inline deployment/CP01CNSM_Deploy.csv:511]: Updates look good

File: deployment/CP02PMCI_Deploy.csv [hunk 2]
```diff
@@ -125,21 +125,21 @@ AR44,,AR52,,CP02PMCI-WFP01-02-DOFSTK000,14,1,2020-06-12T13:40:00,2021-04-14T12:0
...
+AR52,,AR66,,CP02PMCI-WFP01-00-WFPENG000,15,1,2021-04-13T13:35:00,2022-04-24T15:53:00,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGCON-CWFPCP-99101,40.226983,-70.878183,,105,128,,,
+AR52,,AR66,,CP02PMCI-WFP01-01-VEL3DK000,15,1,2021-04-13T13:35:00,2022-04-24T15:53:00,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGINS-VEL3DK-00015,40.226983,-70.878183,,105,128,,,
+AR52,,AR66,,CP02PMCI-WFP01-02-DOFSTK000,15,1,2021-04-13T13:35:00,2022-04-24T15:53:00,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGINS-DOFSTK-02978,40.226983,-70.878183,,105,128,,,
+AR52,,AR66,,CP02PMCI-WFP01-03-CTDPFK000,15,1,2021-04-13T13:35:00,2022-04-24T15:53:00,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGINS-CTDPFK-00111,40.226983,-70.878183,,105,128,,,
+AR52,,AR66,,CP02PMCI-WFP01-04-FLORTK000,15,1,2021-04-13T13:35:00,2022-04-24T15:53:00,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGINS-FLORTK-01029,40.226983,-70.878183,,105,128,,,
+AR52,,AR66,,CP02PMCI-WFP01-05-PARADK000,15,1,2021-04-13T13:35:00,2022-04-24T15:53:00,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGINS-PARADK-20541,40.226983,-70.878183,,105,128,,,
```
Comment (cdobs) [inline deployment/CP02PMCI_Deploy.csv:136]: Looks good

File: deployment/CP02PMCI_Deploy.csv [hunk 3]
```diff
@@ -125,21 +125,21 @@ AR44,,AR52,,CP02PMCI-WFP01-02-DOFSTK000,14,1,2020-06-12T13:40:00,2021-04-14T12:0
...
+AR66,,,,CP02PMCI-SBS01-00-STCENG000,16,1,2022-04-24T13:43:00,,CGMCP-02PMCI-00016,,3717-00004-00001-00002,40.226633,-70.889067,,0,128,,,
+AR66,,,,CP02PMCI-SBS01-01-MOPAK0000,16,1,2022-04-24T13:43:00,,CGMCP-02PMCI-00016,,CGINS-MOPAK0-58484,40.226633,-70.889067,,0,128,,,
+AR66,,,,CP02PMCI-WFP01-00-WFPENG000,16,1,2022-04-24T13:43:00,,CGMCP-02PMCI-00016,CGVEH-CWFPCP-10302,CGCON-CWFPCP-10302,40.226633,-70.889067,,105,128,,,
+AR66,,,,CP02PMCI-WFP01-01-VEL3DK000,16,1,2022-04-24T13:43:00,,CGMCP-02PMCI-00016,CGVEH-CWFPCP-10302,CGINS-VEL3DK-00037,40.226633,-70.889067,,105,128,,,
+AR66,,,,CP02PMCI-WFP01-02-DOFSTK000,16,1,2022-04-24T13:43:00,,CGMCP-02PMCI-00016,CGVEH-CWFPCP-10302,CGINS-DOFSTK-02724,40.226633,-70.889067,,105,128,,,
+AR66,,,,CP02PMCI-WFP01-03-CTDPFK000,16,1,2022-04-24T13:43:00,,CGMCP-02PMCI-00016,CGVEH-CWFPCP-10302,CGINS-CTDPFK-00125,40.226633,-70.889067,,105,128,,,
+AR66,,,,CP02PMCI-WFP01-04-FLORTK000,16,1,2022-04-24T13:43:00,,CGMCP-02PMCI-00016,CGVEH-CWFPCP-10302,CGINS-FLORTK-01119,40.226633,-70.889067,,105,128,,,
+AR66,,,,CP02PMCI-WFP01-05-PARADK000,16,1,2022-04-24T13:43:00,,CGMCP-02PMCI-00016,CGVEH-CWFPCP-10302,CGINS-PARADK-20465,40.226633,-70.889067,,105,128,,,
```
Comment (cdobs) [inline deployment/CP02PMCI_Deploy.csv:145]: looks good

File: deployment/CP02PMCO_Deploy.csv [hunk 4]
```diff
@@ -134,21 +134,21 @@ AR52,,AR61,,CP02PMCO-WFP01-02-DOFSTK000,15,1,2021-04-11T13:44:00,2021-11-10T14:4
...
+AR61,,AR66,,CP02PMCO-WFP01-00-WFPENG000,16,1,2021-11-10T13:15:00,2022-04-23T16:15:00,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGCON-CWFPCP-10305,40.100998,-70.887272,,125,146,,,
+AR61,,AR66,,CP02PMCO-WFP01-01-VEL3DK000,16,1,2021-11-10T13:15:00,2022-04-23T16:15:00,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGINS-VEL3DK-00034,40.100998,-70.887272,,125,146,,,
+AR61,,AR66,,CP02PMCO-WFP01-02-DOFSTK000,16,1,2021-11-10T13:15:00,2022-04-23T16:15:00,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGINS-DOFSTK-02628,40.100998,-70.887272,,125,146,,,
+AR61,,AR66,,CP02PMCO-WFP01-03-CTDPFK000,16,1,2021-11-10T13:15:00,2022-04-23T16:15:00,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGINS-CTDPFK-00128,40.100998,-70.887272,,125,146,,,
+AR61,,AR66,,CP02PMCO-WFP01-04-FLORTK000,16,1,2021-11-10T13:15:00,2022-04-23T16:15:00,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGINS-FLORTK-01101,40.100998,-70.887272,,125,146,,,
+AR61,,AR66,,CP02PMCO-WFP01-05-PARADK000,16,1,2021-11-10T13:15:00,2022-04-23T16:15:00,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGINS-PARADK-20466,40.100998,-70.887272,,125,146,,,
```
Comment (cdobs) [inline deployment/CP02PMCO_Deploy.csv:145]: looks good

File: deployment/CP02PMUI_Deploy.csv [hunk 5]
```diff
@@ -134,12 +134,21 @@ AR44,,,,CP02PMUI-WFP01-02-DOFSTK000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00
...
+AR52,,AR66,,CP02PMUI-SBS01-01-MOPAK0000,16,1,2021-04-12T13:49:00,2022-04-25T15:56:00,CGMCP-02PMUI-00016,,CGINS-MOPAK0-43252,40.365183,-70.7697,,0,95,,,
+AR52,,AR66,,CP02PMUI-WFP01-00-WFPENG000,16,1,2021-04-12T13:49:00,2022-04-25T15:56:00,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGCON-CWFPCP-71401,40.365183,-70.7697,,71,95,,,
+AR52,,AR66,,CP02PMUI-WFP01-01-VEL3DK000,16,1,2021-04-12T13:49:00,2022-04-25T15:56:00,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGINS-VEL3DK-00012,40.365183,-70.7697,,71,95,,,
+AR52,,AR66,,CP02PMUI-WFP01-02-DOFSTK000,16,1,2021-04-12T13:49:00,2022-04-25T15:56:00,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGINS-DOFSTK-00209,40.365183,-70.7697,,71,95,,,
+AR52,,AR66,,CP02PMUI-WFP01-03-CTDPFK000,16,1,2021-04-12T13:49:00,2022-04-25T15:56:00,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGINS-CTDPFK-00093,40.365183,-70.7697,,71,95,,,
+AR52,,AR66,,CP02PMUI-WFP01-04-FLORTK000,16,1,2021-04-12T13:49:00,2022-04-25T15:56:00,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGINS-FLORTK-00830,40.365183,-70.7697,,71,95,,,
+AR52,,AR66,,CP02PMUI-WFP01-05-PARADK000,16,1,2021-04-12T13:49:00,2022-04-25T15:56:00,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGINS-PARADK-20525,40.365183,-70.7697,,71,95,,,
```
Comment (cdobs) [inline deployment/CP02PMUI_Deploy.csv:145]: looks good

File: deployment/CP02PMUI_Deploy.csv [hunk 6]
```diff
@@ -134,12 +134,21 @@ AR44,,,,CP02PMUI-WFP01-02-DOFSTK000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00
...
+AR66,,,,CP02PMUI-SBS01-00-STCENG000,17,1,2022-04-25T13:44:00,,CGMCP-02PMUI-00017,,3717-00004-00001-00008,40.364917,-70.780483,,0,95,,,
+AR66,,,,CP02PMUI-SBS01-01-MOPAK0000,17,1,2022-04-25T13:44:00,,CGMCP-02PMUI-00017,,CGINS-MOPAK0-11345,40.364917,-70.780483,,0,95,,,
+AR66,,,,CP02PMUI-WFP01-00-WFPENG000,17,1,2022-04-25T13:44:00,,CGMCP-02PMUI-00017,CGVEH-CWFPCP-10303,CGCON-CWFPCP-10303,40.364917,-70.780483,,71,95,,,
+AR66,,,,CP02PMUI-WFP01-01-VEL3DK000,17,1,2022-04-25T13:44:00,,CGMCP-02PMUI-00017,CGVEH-CWFPCP-10303,CGINS-VEL3DK-00036,40.364917,-70.780483,,71,95,,,
+AR66,,,,CP02PMUI-WFP01-02-DOFSTK000,17,1,2022-04-25T13:44:00,,CGMCP-02PMUI-00017,CGVEH-CWFPCP-10303,CGINS-DOFSTK-02725,40.364917,-70.780483,,71,95,,,
+AR66,,,,CP02PMUI-WFP01-03-CTDPFK000,17,1,2022-04-25T13:44:00,,CGMCP-02PMUI-00017,CGVEH-CWFPCP-10303,CGINS-CTDPFK-00126,40.364917,-70.780483,,71,95,,,
+AR66,,,,CP02PMUI-WFP01-04-FLORTK000,17,1,2022-04-25T13:44:00,,CGMCP-02PMUI-00017,CGVEH-CWFPCP-10303,CGINS-FLORTK-01118,40.364917,-70.780483,,71,95,,,
+AR66,,,,CP02PMUI-WFP01-05-PARADK000,17,1,2022-04-25T13:44:00,,CGMCP-02PMUI-00017,CGVEH-CWFPCP-10303,CGINS-PARADK-20468,40.364917,-70.780483,,71,95,,,
```
Comment (cdobs) [inline deployment/CP02PMUI_Deploy.csv:154]: looks good

File: deployment/CP02PMUO_Deploy.csv [hunk 7]
```diff
@@ -152,21 +152,21 @@ AR52,,AR61,,CP02PMUO-WFP01-02-DOFSTK000,17,1,2021-04-10T13:41:00,2021-11-13T12:3
...
+AR66,,,,CP02PMUO-SBS01-00-STCENG000,19,1,2022-04-22T14:22:00,,CGMCP-02PMUO-00019,,3717-00004-00001-00010,39.939333,-70.770183,,0,453,,,
+AR66,,,,CP02PMUO-SBS01-01-MOPAK0000,19,1,2022-04-22T14:22:00,,CGMCP-02PMUO-00019,,CGINS-MOPAK0-70027,39.939333,-70.770183,,0,453,,,
+AR66,,,,CP02PMUO-WFP01-00-WFPENG000,19,1,2022-04-22T14:22:00,,CGMCP-02PMUO-00019,CGVEH-CWFPCP-10304,CGCON-CWFPCP-10304,39.939333,-70.770183,,429,453,,,
+AR66,,,,CP02PMUO-WFP01-01-VEL3DK000,19,1,2022-04-22T14:22:00,,CGMCP-02PMUO-00019,CGVEH-CWFPCP-10304,CGINS-VEL3DK-00033,39.939333,-70.770183,,429,453,,,
+AR66,,,,CP02PMUO-WFP01-02-DOFSTK000,19,1,2022-04-22T14:22:00,,CGMCP-02PMUO-00019,CGVEH-CWFPCP-10304,CGINS-DOFSTK-02726,39.939333,-70.770183,,429,453,,,
+AR66,,,,CP02PMUO-WFP01-03-CTDPFK000,19,1,2022-04-22T14:22:00,,CGMCP-02PMUO-00019,CGVEH-CWFPCP-10304,CGINS-CTDPFK-00127,39.939333,-70.770183,,429,453,,,
+AR66,,,,CP02PMUO-WFP01-04-FLORTK000,19,1,2022-04-22T14:22:00,,CGMCP-02PMUO-00019,CGVEH-CWFPCP-10304,CGINS-FLORTK-01100,39.939333,-70.770183,,429,453,,,
+AR66,,,,CP02PMUO-WFP01-05-PARADK000,19,1,2022-04-22T14:22:00,,CGMCP-02PMUO-00019,CGVEH-CWFPCP-10304,CGINS-PARADK-20582,39.939333,-70.770183,,429,453,,,
```
Comment (cdobs) [inline deployment/CP02PMUO_Deploy.csv:172]: looks good

File: deployment/CP02PMUO_Deploy.csv [hunk 8]
```diff
@@ -152,21 +152,21 @@ AR52,,AR61,,CP02PMUO-WFP01-02-DOFSTK000,17,1,2021-04-10T13:41:00,2021-11-13T12:3
...
+AR61,,AR66,,CP02PMUO-WFP01-00-WFPENG000,18,1,2021-11-09T20:23:00,2022-04-22T16:02:00,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGCON-CWFPCP-10301,39.941533,-70.780857,,425,449,,,
+AR61,,AR66,,CP02PMUO-WFP01-01-VEL3DK000,18,1,2021-11-09T20:23:00,2022-04-22T16:02:00,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGINS-VEL3DK-00035,39.941533,-70.780857,,425,449,,,
+AR61,,AR66,,CP02PMUO-WFP01-02-DOFSTK000,18,1,2021-11-09T20:23:00,2022-04-22T16:02:00,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGINS-DOFSTK-02708,39.941533,-70.780857,,425,449,,,
+AR61,,AR66,,CP02PMUO-WFP01-03-CTDPFK000,18,1,2021-11-09T20:23:00,2022-04-22T16:02:00,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGINS-CTDPFK-00124,39.941533,-70.780857,,425,449,,,
+AR61,,AR66,,CP02PMUO-WFP01-04-FLORTK000,18,1,2021-11-09T20:23:00,2022-04-22T16:02:00,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGINS-FLORTK-01117,39.941533,-70.780857,,425,449,,,
+AR61,,AR66,,CP02PMUO-WFP01-05-PARADK000,18,1,2021-11-09T20:23:00,2022-04-22T16:02:00,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGINS-PARADK-20610,39.941533,-70.780857,,425,449,,,
```
Comment (cdobs) [inline deployment/CP02PMUO_Deploy.csv:163]: looks good

File: deployment/CP03ISPM_Deploy.csv [hunk 9]
```diff
@@ -35,12 +35,12 @@ AR48,,AR52,,CP03ISPM-WFP01-02-DOFSTK000,4,1,2020-10-31T22:02:00,2021-04-12T16:31
...
+AR61,,AR66,,CP03ISPM-SBS01-01-MOPAK0000,5,1,2021-11-11T13:27:00,2022-04-25T19:20:00,CGMCP-03ISPM-00005,,CGINS-MOPAK0-58481,40.362005,-70.878502,,0,93,,,
+AR61,,AR66,,CP03ISPM-WFP01-00-WFPENG000,5,1,2021-11-11T13:27:00,2022-04-25T19:20:00,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGCON-CWFPCP-30001,40.362005,-70.878502,,69,93,,,
+AR61,,AR66,,CP03ISPM-WFP01-01-VEL3DK000,5,1,2021-11-11T13:27:00,2022-04-25T19:20:00,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGINS-VEL3DK-00468,40.362005,-70.878502,,69,93,,,
+AR61,,AR66,,CP03ISPM-WFP01-02-DOFSTK000,5,1,2021-11-11T13:27:00,2022-04-25T19:20:00,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGINS-DOFSTK-03506,40.362005,-70.878502,,69,93,,,
+AR61,,AR66,,CP03ISPM-WFP01-03-CTDPFK000,5,1,2021-11-11T13:27:00,2022-04-25T19:20:00,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGINS-CTDPFK-00184,40.362005,-70.878502,,69,93,,,
+AR61,,AR66,,CP03ISPM-WFP01-04-FLORTK000,5,1,2021-11-11T13:27:00,2022-04-25T19:20:00,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGINS-FLORTK-01537,40.362005,-70.878502,,69,93,,,
+AR61,,AR66,,CP03ISPM-WFP01-05-PARADK000,5,1,2021-11-11T13:27:00,2022-04-25T19:20:00,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGINS-PARADK-20608,40.362005,-70.878502,,69,93,,,
```
Comment (cdobs) [inline deployment/CP03ISPM_Deploy.csv:46]: looks good

File: deployment/CP03ISSM_Deploy.csv [hunk 10]
```diff
@@ -403,37 +403,37 @@ AR52,,AR61,,CP03ISSM-SBD11-06-METBKA000,13,1,2021-04-02T20:47:00,2021-11-11T19:3
...
+AR61,,AR66,,CP03ISSM-SBD11-00-DCLENG000,14,1,2021-11-01T14:10:00,2022-04-13T21:24:00,CGMCP-03ISSM-00014,,CGCON-BDCL11-50004,40.3646,-70.888252,,0,93,,,
+AR61,,AR66,,CP03ISSM-SBD11-01-MOPAK0000,14,1,2021-11-01T14:10:00,2022-04-13T21:24:00,CGMCP-03ISSM-00014,,CGINS-MOPAK0-11839,40.3646,-70.888252,,0,93,,,
+AR61,,AR66,,CP03ISSM-SBD11-02-HYDGN0000,14,1,2021-11-01T14:10:00,2022-04-13T21:24:00,CGMCP-03ISSM-00014,,CGINS-HYDGN0-41692,40.3646,-70.888252,,0,93,,,
+AR61,,AR66,,CP03ISSM-SBD11-06-METBKA000,14,1,2021-11-01T14:10:00,2022-04-13T21:24:00,CGMCP-03ISSM-00014,,CGINS-METLGR-00033,40.3646,-70.888252,,0,93,,,
+AR61,,AR66,,CP03ISSM-SBD12-00-DCLENG000,14,1,2021-11-01T14:10:00,2022-04-13T21:24:00,CGMCP-03ISSM-00014,,CGCON-BDCL12-50004,40.3646,-70.888252,,0,93,,,
+AR61,,AR66,,CP03ISSM-SBD12-03-HYDGN0000,14,1,2021-11-01T14:10:00,2022-04-13T21:24:00,CGMCP-03ISSM-00014,,CGINS-HYDGN0-49868,40.3646,-70.888252,,0,93,,,
+AR61,,AR66,,CP03ISSM-SBD12-04-PCO2AA000,14,1,2021-11-01T14:10:00,2022-04-13T21:24:00,CGMCP-03ISSM-00014,,CGINS-PCO2AA-33182,40.3646,-70.888252,,0,93,,,
```
Comment (cdobs) [inline deployment/CP03ISSM_Deploy.csv:436]: looks good

File: deployment/CP05MOAS-GL379_Deploy.csv [hunk 11]
```diff
@@ -77,9 +77,9 @@ TS20210903,,TS20211214,,CP05MOAS-GL379-02-FLORTM000,13,1,2021-09-03T22:51:00,202
...
\ No newline at end of file
+TS20220104,,AR66,,CP05MOAS-GL379-00-ENG000000,14,1,2022-01-04T20:52:00,2022-04-21T23:03:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.94263,-70.38205,,1000,1000,SS-2,,
+TS20220104,,AR66,,CP05MOAS-GL379-01-ADCPAM000,14,1,2022-01-04T20:52:00,2022-04-21T23:03:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-48878,39.94263,-70.38205,,1000,1000,SS-2,,
+TS20220104,,AR66,,CP05MOAS-GL379-02-FLORTM000,14,1,2022-01-04T20:52:00,2022-04-21T23:03:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-03207,39.94263,-70.38205,,1000,1000,SS-2,,
+TS20220104,,AR66,,CP05MOAS-GL379-03-CTDGVM000,14,1,2022-01-04T20:52:00,2022-04-21T23:03:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09362,39.94263,-70.38205,,1000,1000,SS-2,,
+TS20220104,,AR66,,CP05MOAS-GL379-04-DOSTAM000,14,1,2022-01-04T20:52:00,2022-04-21T23:03:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00123,39.94263,-70.38205,,1000,1000,SS-2; top mount,,
+TS20220104,,AR66,,CP05MOAS-GL379-05-PARADM000,14,1,2022-01-04T20:52:00,2022-04-21T23:03:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50204,39.94263,-70.38205,,1000,1000,SS-2,,
```
Comment (cdobs) [inline deployment/CP05MOAS-GL379_Deploy.csv:85]: looks good

File: deployment/CP04OSPM_Deploy.csv [hunk 12]
```diff
@@ -118,21 +118,21 @@ AR52,,AR61,,CP04OSPM-WFP01-02-DOFSTK000,14,1,2021-04-09T14:05:00,2021-11-13T17:0
...
+AR61,,AR66,,CP04OSPM-WFP01-00-WFPENG000,15,1,2021-11-09T14:59:00,2022-04-21T16:42:00,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGCON-CWFPCP-10303,39.943353,-70.887295,,414,435,,,
+AR61,,AR66,,CP04OSPM-WFP01-01-VEL3DK000,15,1,2021-11-09T14:59:00,2022-04-21T16:42:00,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGINS-VEL3DK-00036,39.943353,-70.887295,,414,435,,,
+AR61,,AR66,,CP04OSPM-WFP01-02-DOFSTK000,15,1,2021-11-09T14:59:00,2022-04-21T16:42:00,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGINS-DOFSTK-02725,39.943353,-70.887295,,414,435,,,
+AR61,,AR66,,CP04OSPM-WFP01-03-CTDPFK000,15,1,2021-11-09T14:59:00,2022-04-21T16:42:00,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGINS-CTDPFK-00126,39.943353,-70.887295,,414,435,,,
+AR61,,AR66,,CP04OSPM-WFP01-04-FLORTK000,15,1,2021-11-09T14:59:00,2022-04-21T16:42:00,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGINS-FLORTK-01118,39.943353,-70.887295,,414,435,,,
+AR61,,AR66,,CP04OSPM-WFP01-05-PARADK000,15,1,2021-11-09T14:59:00,2022-04-21T16:42:00,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGINS-PARADK-20468,39.943353,-70.887295,,414,435,,,
```
Comment (cdobs) [inline deployment/CP04OSPM_Deploy.csv:129]: looks good

File: deployment/CP04OSPM_Deploy.csv [hunk 13]
```diff
@@ -118,21 +118,21 @@ AR52,,AR61,,CP04OSPM-WFP01-02-DOFSTK000,14,1,2021-04-09T14:05:00,2021-11-13T17:0
...
+AR66,,,,CP04OSPM-SBS01-00-STCENG000,16,1,2022-04-21T14:28:00,,CGMCP-04OSPM-00016,,3717-00004-00001-00007,39.936,-70.879417,,0,457,,,
+AR66,,,,CP04OSPM-SBS11-02-MOPAK0000,16,1,2022-04-21T14:28:00,,CGMCP-04OSPM-00016,,CGINS-MOPAK0-11347,39.936,-70.879417,,0,457,,,
+AR66,,,,CP04OSPM-WFP01-00-WFPENG000,16,1,2022-04-21T14:28:00,,CGMCP-04OSPM-00016,CGVEH-CWFPCP-99105,CGCON-CWFPCP-99105,39.936,-70.879417,,431,457,,,
+AR66,,,,CP04OSPM-WFP01-01-VEL3DK000,16,1,2022-04-21T14:28:00,,CGMCP-04OSPM-00016,CGVEH-CWFPCP-99105,CGINS-VEL3DK-00212,39.936,-70.879417,,431,457,,,
+AR66,,,,CP04OSPM-WFP01-02-DOFSTK000,16,1,2022-04-21T14:28:00,,CGMCP-04OSPM-00016,CGVEH-CWFPCP-99105,CGINS-DOFSTK-03370,39.936,-70.879417,,431,457,,,
+AR66,,,,CP04OSPM-WFP01-03-CTDPFK000,16,1,2022-04-21T14:28:00,,CGMCP-04OSPM-00016,CGVEH-CWFPCP-99105,CGINS-CTDPFK-00169,39.936,-70.879417,,431,457,,,
+AR66,,,,CP04OSPM-WFP01-04-FLORTK000,16,1,2022-04-21T14:28:00,,CGMCP-04OSPM-00016,CGVEH-CWFPCP-99105,CGINS-FLORTK-01466,39.936,-70.879417,,431,457,,,
+AR66,,,,CP04OSPM-WFP01-05-PARADK000,16,1,2022-04-21T14:28:00,,CGMCP-04OSPM-00016,CGVEH-CWFPCP-99105,CGINS-PARADK-20542,39.936,-70.879417,,431,457,,,
```
Comment (cdobs) [inline deployment/CP04OSPM_Deploy.csv:138]: looks good

**Review by cdobs — APPROVED** [review]

**Comment by rtravis-whoi** [comment]
[Pioneer 18 AM Updates [DT-152]](https://trello.com/c/zFi7BfCv/152-pioneer-18-am-updates-dt-152)

**Pull request merged**